addons/google_calendar/models/calendar_recurrence_rule.py

```diff
@@ -185,3 +185,10 @@ def _notify_attendees(self):
         partners = recurrences.base_event_id.partner_ids
         if partners:
             self.env['calendar.alarm_manager']._notify_next_alarm(partners.ids)
+
+    def _get_event_user(self):
+        self.ensure_one()
+        event = self._get_first_event()
+        if event:
+            return event._get_event_user()
+        return self.env.user
```

Comment:
```suggestion
        return self._get_first_event()._get_event_user() or self.env.user
```
`_get_first_event` always returns a `calendar.event` recordset, (can be an empty one)

Reply:
Thanks for your review.
It would raise an error if it is an empty recordset. (because of ensure_one())